client/app/bundles/buttons/button.scss

```diff
@@ -0,0 +1,79 @@
+@import "base.scss";
+
+
+button:hover{
+         
+}
+.buttonFilledM{
+         fontSize: 18px;
+         color: #6D0839;
+         font-weight: bold;
+         background-color: #FFFFFF;
+         padding-top: 17px;
+         padding-bottom: 16px;
+         padding-left: 15px;
+         padding-right: 15px;
+         width: 203px;
```

Comment:
Sorry for not catching this before, but let's make sure we don't give these buttons explicit dimensions! If you could keep the spacing between the text and the edges of the container consisting using padding, that would be perfect.